### PR TITLE
Remove babel helpers

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md not updated, noop refactoring

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "parse5": "^2.2.3",
     "plylog": "^0.4.0",
     "polymer-analyzer": "^2.0.0",
-    "polymer-build": "^1.3.0",
+    "polymer-build": "1.4.0",
```
Comment (usergenic) [inline package.json]: Should we be pegging specific dependencies or can we ^1.4.0 this?

Comment (rictic) [inline package.json]: +1, did `yarn add polymer-build@1.4.0` do this?

File: yarn.lock [hunk 2]
```diff
@@ -5382,7 +5382,28 @@ polymer-analyzer@^2.0.0, polymer-analyzer@^2.0.1:
...
+    parse5 "^2.2.2"
+    plylog "^0.5.0"
+    polymer-analyzer "^2.0.1"
+    polymer-bundler "2.0.0-pre.18"
+    polymer-project-config "^3.0.0"
+    sw-precache "^5.1.1"
+    vinyl "^1.1.1"
+    vinyl-fs "^2.4.3"
+
+polymer-build@^1.1.0:
```
Comment (usergenic) [inline yarn.lock]: We have multiple polymer-builds in here?

Comment (rictic) [inline yarn.lock]: Huh, weird. Updated. Can confirm that the updated yarn.lock only results in one version of polymer-build ending up in node_modules

**Comment by usergenic** [comment]
also LGTM

**Pull request merged**